luigi/contrib/mrjob.py

```diff
@@ -0,0 +1,91 @@
+# Some awesome license
+
+# import mrjob
+from __future__ import absolute_import
+
+from cStringIO import StringIO
+from luigi.task import flatten
+import logging
+import luigi
+import luigi.s3
+from datetime import datetime
+
+
+logger = logging.getLogger('luigi-interface')
+
+
+class MRJobTask(luigi.Task):
+    job_class = None
+    job_args = luigi.Parameter(default=None)
+    job_options = {}
+    job_results = None
+    s3_output_path = None
+    misc_options = {}
+
+    def run_inline(self, job):
+        print 'gothereherhhe', self
```

Comment:
@sambodanis i am still trying to unravel the meanings in the depth of this sentence ;-)